api/app/Http/Controllers/CollectionController.php

```diff
@@ -124,10 +125,146 @@ public function showOneCollection($id, Request $request){
+	 * 					maxLength=45
+	 * 				),
+	 *  				@OA\Property(
+	 * 					property="managementType",
+	 * 					type="string",
+	 * 					description="Use 'Snapshot' when there is a separate in-house database maintained in the collection and the dataset within the Symbiota portal is only a periodically updated snapshot of the central database. A 'Live Data' dataset is when the data is managed directly within the portal and the central database is the portal data",
+	 * 					maxLength=45
+	 * 				),
+	 *  				@OA\Property(
+	 * 					property="publicEdits",
+	 * 					type="integer",
+	 * 					description="The option to enable public edits (1 for yes, 0 for no)",
+	 * 					maxLength=1
+	 * 				),
+	 *  				@OA\Property(
+	 * 					property="rightsHolder",
+	 * 					type="string",
+	 * 					description="The organization or person managing or owning the rights of the resource. For more details, see Darwin Core definition",
+	 * 					maxLength=250
+	 * 				),
+	 *  				@OA\Property(
+	 * 					property="rights",
+	 * 					type="string",
+	 * 					description="Information or a URL link to page with details explaining how one can use the data. See Darwin Core definition",
+	 * 					maxLength=250
+	 * 				),
+	 *  				@OA\Property(
+	 * 					property="accessRights",
+	 * 					type="string",
+	 * 					description="Information or a URL link to page with details explaining how one can use the data. See Darwin Core definition",
+	 * 					maxLength=1000
+	 * 				),
+	 *  				@OA\Property(
+	 * 					property="sortSeq",
+	 * 					type="string",
+	 * 					description="Leave this field empty if you want the collections to sort alphabetically (default)",
+	 * 					maxLength=10
+	 * 				),
+	 *  				@OA\Property(
+	 * 					property="icon",
+	 * 					type="string",
+	 * 					description="URL of an image icon representing the collection. The URL path can be absolute or relative. The use of icons are optional",
+	 * 					maxLength=250
+	 * 				),
+	 * 			)
+	 * 		)
+	 * 	),
+	 *	 @OA\Response(
+	 *		 response="200",
+	 *		 description="Returns full JSON object of the of collection that was created"
+	 *	 ),
+	 *	 @OA\Response(
+	 *		 response="401",
+	 *		 description="Unauthorized",
+	 *	 ),
+	 * )
+	 */
 	public function create(Request $request){
-		//$collection = Collection::create($request->all());
-		//return response()->json($collection, 201);
-	}
+		// if($user = $this->authenticate($request)){
+			// @TODO make colleciton GUID?
+			try {
+				$collection = Collection::create($request->all());
+				$collectionStats = CollectionStats::create([
+					'collid' => $collection->collID,
+					'recordcnt' => 0,
+					// 'uploadedby' => $GLOBALS['USERNAME']
```

Comment:
It can't access the $GLOBALS variable, and I'm not quite sure why or how to resolve.